src/Features/Core/Portable/ChangeSignature/AbstractChangeSignatureService.cs

```diff
@@ -1054,5 +1054,21 @@ protected static bool IsParamsArrayExpandedHelper(ISymbol symbol, int argumentCo
 
             return false;
         }
+
+        protected static int GetParameterIndexFromInvocationArgument(SyntaxNode argument, Document document, SemanticModel semanticModel, CancellationToken cancellationToken)
+        {
+            var semanticFacts = document.GetRequiredLanguageService<ISemanticFactsService>();
+            var parameter = semanticFacts.FindParameterForArgument(semanticModel, argument, cancellationToken);
+            var parameterIndex = parameter.Ordinal;
+
+            if (parameter.ContainingSymbol is IMethodSymbol methodSymbol && methodSymbol.IsExtensionMethod)
```

Comment:
does this work if the caller is calling it in static form, vs instance form?  do you have tests for that?